### PR TITLE
Moving build to dotnet

### LINKED ISSUE #18: Change to build to use dotnet

### PR DESCRIPTION
**Pull request by moozzyk**
Fixes #18

### CODE REVIEW

**Comment by moozzyk** [comment]
@victorhurdugaci

**Comment by moozzyk** [comment]
@victorhurdugaci :up::date:

File: NuGet.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
-    <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v2" />
```
Comment (victorhurdugaci) [inline NuGet.config]: v3

**Comment by victorhurdugaci** [comment]
:shipit:

**Pull request merged**